utilities/blob_db/blob_db_impl.cc

```diff
@@ -1123,11 +1127,17 @@ Status BlobDBImpl::PutBlobValue(const WriteOptions& /*options*/,
 
 Slice BlobDBImpl::GetCompressedSlice(const Slice& raw,
                                      std::string* compression_output) const {
-  if (bdb_options_.compression == kNoCompression) {
+  return GetCompressedSlice(raw, bdb_options_.compression, compression_output);
```

Comment:
I was wondering why this overload was needed? We exclusively use `bdb_options_.compression`, right?

Reply:
You are right, we don't need this anymore.